service/src/test/java/tds/student/web/endpoints/StudentControllerIntegrationTests.java

```diff
@@ -23,12 +23,12 @@ public void shouldReturnStudent() {
         given()
             .accept(ContentType.JSON)
         .when()
-            .get(String.format("/student/%d", 1))
+            .get(String.format("/students/%d", 1))
         .then()
             .contentType(ContentType.JSON)
             .statusCode(200)
             .body("student.id", equalTo(1))
-            .body("student.studentId", equalTo("adv001"))
+            .body("student.externalStudentId", equalTo("adv001"))
             .body("student.stateCode", equalTo("CA"))
             .body("student.clientName", equalTo("SBAC_PT"));
```

Comment:
Certainly not a huge deal, but might be worthwhile to verify the self link has been built correctly.